Announce release policy

waiting-for-dev opened the pull request:
Blog post to announce our new [release policy](https://solidus.io/release_policy).

Creating a draft, as I need to ask @mfrecchiami for a picture, but you can already review the content.

waiting-for-dev commented:
I added the image. This is ready for review 🙂

elia commented on source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown:
```diff
@@ -0,0 +1,36 @@
+We want to improve there. Because of that, we discussed and agreed on a release
+policy that we hope will help the community prepare for Solidus upgrades. As
+always in Open Source, resources are limited, but we're confident that this
+self-inflicted pressure will force us to keep pace with your expectations.
```
thoughts on `force us` → `help us`?

waiting-for-dev replied:
I like it. Less aggressive. Done!

elia commented on source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown:
```diff
@@ -0,0 +1,36 @@
+- Major releases should not be more than two years apart.
+- Patch releases at every backported fix.
+
+Share your ideas. Please, don't think twice about reaching us on our [Slack
```
not sure but worth checking, `reaching us` → `reaching out to us`

waiting-for-dev replied:
I think you're right. Done!

elia reviewed (approved):
👍 just left a couple of optional nits

waiting-for-dev commented on source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown:
```diff
@@ -0,0 +1,36 @@
+
+Share your ideas. Please, don't think twice about reaching us on our [Slack
+workspace](http://slack.solidus.io/), and let us know any suggestions or
+concerns you might experience. We're here to make your experience with the most
```
Yeah. Also done :slightly_smiling_face:

waiting-for-dev commented:
> +1 just left a couple of optional nits

Thanks for your suggestions, @elia. I incorporated all of them. I also updated the publication date to today.
